pyproject.toml

```diff
@@ -155,7 +168,6 @@ exclude = [
     'rule_based_profiler/parameter_builder/value_set_multi_batch_parameter_builder\.py',  # 2
     'rule_based_profiler/parameter_container\.py',  # 7
     'rule_based_profiler/rule_based_profiler\.py',  # 40
-    'tasks\.py',  # Excluded from type checking the rest of the library, may be added in a separate step.
```

Comment:
@anthonyburdi why were we ignoring this file?
I'm not getting any errors in the CI.